modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -442,9 +442,9 @@ public void loginWithKeycard(final String whisperPrivateKey, final String encryp
         Runnable r = new Runnable() {
             @Override
             public void run() {
-                  String result = Statusgo.loginWithKeycard(whisperPrivateKey, encryptionPublicKey);
+                  //String result = Statusgo.loginWithKeycard(whisperPrivateKey, encryptionPublicKey);
```

Comment:
hows this related?

Reply:
@cammellos you can just rebase I already fixed that